tasks/ssh.yml

```diff
@@ -0,0 +1,6 @@
+- name: Create default SSH key pair
+  user:
+    name: "{{ www_user }}"
```

Comment:
I think we should use root and make sure the generated files have strict permissions (eg: 0600) so they're readable only by root. Using www-data means that in case of a security breach in the web application, if an attacker gains access to part of the filesystem and is able to read the home directory, it will be able to read the private key, which can have potential very negative consequences

Reply:
The permissions are set automatically by the `user` module. 

In the update implementation, I have created separate task for reading generated keys. I thought running a script with escalated privileges was not a good idea.